package.json

```diff
@@ -2,7 +2,7 @@
   "name": "suomifi-design-system",
   "private": true,
   "description": "Suomi.fi Design System",
-  "version": "0.1.0",
+  "version": "0.1.1",
```

Comment:
Shouldn't this still be `0.1.0`? 
Currently, at the time of writing this, the develop-branch has `0.1.0`

Reply:
Something went wrong with rebase, should be fixed now.